dnf/cli/output.py

```diff
@@ -1676,44 +1690,32 @@ def _simple_pkg(pkg, prefix_len, was_installed=False, highlight=False,
                 print(_("Additional non-default information stored: %d") % \
                           len(non_default))
 
-        if old.trans_with:
-            # This is _possible_, but not common
+        perf_with = old.performed_with()
+        if perf_with:
             print(_("Transaction performed with:"))
-            pkg_max_len = max((len(str(hpkg)) for hpkg in old.trans_with))
-        for hpkg in old.trans_with:
-            _simple_pkg(hpkg, 4, was_installed=True, pkg_max_len=pkg_max_len)
+        max_len = 0
+        for spkg in perf_with:
+            if len(spkg.nevra) > max_len:
+                max_len = len(spkg.nevra)
+        for spkg in perf_with:
+            _simple_pkg(spkg, 4, was_installed=True, pkg_max_len=max_len)
+
         print(_("Packages Altered:"))
+
         self.historyInfoCmdPkgsAltered(old, pats)
 
-        if old.trans_skip:
-            print(_("Packages Skipped:"))
-            pkg_max_len = max((len(str(hpkg)) for hpkg in old.trans_skip))
-        for hpkg in old.trans_skip:
-            #  Don't show the repo. here because we can't store it as they were,
-            # by definition, not installed.
-            _simple_pkg(hpkg, 4, pkg_max_len=pkg_max_len, show_repo=False)
-
-        if old.rpmdb_problems:
-            print(_("Rpmdb Problems:"))
```

Comment:
`trans_skip` and `rpmdb_probles` were dropped in swdb design - they aren't used anymore